qutebrowser/browser/network/pac.py

```diff
@@ -254,6 +254,7 @@ def __init__(self, url, parent=None):
     def __eq__(self, other):
         # pylint: disable=protected-access
         return self._pac_url == other._pac_url
+        # pylint: enable=protected-access
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.